lib/WP/Functions.php

```diff
@@ -154,7 +158,10 @@ function isRtl() {
   function currentUserCan() {
     return call_user_func_array('current_user_can', func_get_args());
   }
-
+  
+  /**
+   * @return \WP_Role | null
```

Comment:
In other places we don't have whitespaces around `|`.

Reply:
I will change it